src/worker/tasks/sendTransactionWorker.ts

```diff
@@ -81,11 +81,48 @@ const handler: Processor<any, void, string> = async (job: Job<string>) => {
   // null = the transaction attemped to resend but was not needed. Ignore.
   // A thrown exception indicates a retry-able error occurred (e.g. RPC outage).
   let resultTransaction: SentTransaction | ErroredTransaction | null;
-  if (transaction.status === "queued") {
+
+  let populatedTransaction:
+    | Awaited<ReturnType<typeof toSerializableTransaction>>
+    | undefined;
+
+  const chain = await getChain(transaction.chainId);
+  // Populate the transaction to resolve gas values.
```

Comment:
Btw if gas value is passed here (make sure it's the 'gas' prop, not 'gasLimit' - then there will be no gas estimation and therefore no simulation.

I think that's actually what people expect in this case though, it's a way to 'force' the tx onchain. 

Lmk if that's fine

Reply:
yep, this is what we expect